src/main/resources/scripts/windowsInstallJavaScript.ps1

```diff
@@ -1,6 +1,6 @@
 Set-ExecutionPolicy Bypass -Scope Process -Force; [System.Net.ServicePointManager]::SecurityProtocol = [System.Net.ServicePointManager]::SecurityProtocol -bor 3072; iex ((New-Object System.Net.WebClient).DownloadString('https://chocolatey.org/install.ps1'))
 
-choco install -y adoptopenjdk11
+choco install -y temurin17
```

Comment:
Looks like these scripts were broken by chocolatey 2, I'm testing a fix atm

Reply:
I had considered switching this to use winget:

```bash
winget install EclipseAdoptium.Temurin.17.JDK
```

Reply:
winget is only available from window server 2022+ as far as I remember

Reply:
The alternative would be to use the Adoptium API directly

```ps
# Set the URL for the Adoptium API
$downloadUrl = "https://api.adoptium.net/v3/installer/latest/17/ga/windows/x64/jdk/hotspot/normal/eclipse"

# Specify the download location
$destination = "$env:TEMP\temurin17.msi"

# Download the JDK installer
Invoke-WebRequest -Uri $downloadUrl -OutFile $destination

# Install the JDK
Start-Process -FilePath msiexec.exe -ArgumentList "/i", $destination, "/quiet", "/norestart" -Wait

# Clean up the installer file
Remove-Item $destination
```

Reply:
Probably ok but for now I've pinned chocolatey to 1.x and it worked:

<img width="666" alt="image" src="https://github.com/user-attachments/assets/c5e9beb6-4104-47ea-a33f-2a70b4cfddc1">